### PR TITLE
ros2 port fixup

### PR DESCRIPTION
**Pull request by mlautman**
adds runtime and removes missing pkg

### CODE REVIEW

**Comment by mlautman** [comment]
@anasarrak and @ahcorde

File: package.xml [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
+
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
+
   <build_depend>rosidl_default_generators</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
 
-  <exec_depend>message_generation</exec_depend> <!-- provide message generation to downstream packages -->
-  <exec_depend>message_runtime</exec_depend>
-  <exec_depend>std_msgs</exec_depend>
```
Comment (anasarrak) [inline package.xml:24]: Why remove the ``<exec_depend>``?

Comment (mlautman) [reply package.xml:24]: `exec_depend` is redundant with `depend`. In general if a dependency is both an `exec_depend` and a `build_depend` I prefer to use `depend` to keep the code more compact

**Comment by mlautman** [comment]
@ahcorde

**Review by anasarrak — APPROVED** [review]

**Review by ahcorde — APPROVED** [review]

**Pull request merged**